pkg/operator/sync.go

```diff
@@ -1814,3 +1873,17 @@ func (optr *Operator) getImageRegistryPullSecrets() ([]byte, error) {
 
 	return nil, nil
 }
+
+func cmToData(cm *corev1.ConfigMap, key string) ([]byte, error) {
```

Comment:
question: Is there a way that the functionality provided by this function could be deduplicated with the one that's in `pkg/daemon`?

If not or it's too much work, feel free to ignore.